apps/src/code-studio/components/progress/ScriptOverview.jsx

```diff
@@ -60,7 +60,8 @@ class ScriptOverview extends React.Component {
     scriptHasLockableStages: PropTypes.bool.isRequired,
     scriptAllowsHiddenStages: PropTypes.bool.isRequired,
     hiddenStageState: PropTypes.object,
-    selectedSectionId: PropTypes.string
+    selectedSectionId: PropTypes.string,
+    userId: PropTypes.number
```

Comment:
I'm using `userId` rather than `studentId` (which is in the `StudentFeedbackNotification`)  because the user viewing the course or unit overview page could be `userType == student` OR `userType == teacher`. "Student" in the context of these notifications really means "member of another teacher's section".